src/libraries/Microsoft.Bcl.Memory/src/Polyfills/System.Numerics.BitOperations.netstandard20.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// Contains a polyfill implementation of System.Numerics.BitOperations that works on netstandard2.0.
+// Implementation copied from:
+// https://github.com/dotnet/runtime/blob/82ab89241b90ca3d64b22971f3a1e248da72828a/src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs
+//
+// Some routines inspired by the Stanford Bit Twiddling Hacks by Sean Eron Anderson:
+// http://graphics.stanford.edu/~seander/bithacks.html
+
+namespace System.Numerics
+{
+    internal static class BitOperations
+    {
+        // C# no-alloc optimization that directly wraps the data section of the dll (similar to string constants)
+        // https://github.com/dotnet/roslyn/pull/24621
+
+        private static ReadOnlySpan<byte> TrailingZeroCountDeBruijn => // 32
```

Comment:
nit: this can be a local in TrailingZeroCount (single use). Slightly better on [netfx as well](https://sharplab.io/#v2:EYLgHgbALANALiAhgZwLYB8ACAmAjAWAChMAGAAk1wDoAlAVwDs4BLVAUyoEkm2AnAewAOAZT4A3ZgGM2yANxEimAMwVsZAMJEA3kTJ6yg3szGI4bCrghkabRABMA8gwA2AT2GDEDADzBXZgD4yABVeRGZnZgYAcwAtPn51fkY4ABE2ACFeOmYAKwYyAF4ggHoSsiVsXX0AbWr9fRISGDISXBbsAA4WkmwOgE4W3FgybBGSJRh6horm0b7RudwAVg7VslwAdh7x7umGpXbR7Y3J0bO8IcGNiB7t/f1sW42F3G7W59wjknXcOZJ+tMALryQjTZRkfjAXJsSRwMgAWQAFABKaY6QgzfSYTaItiofi8VwIxC8ZAAC0QzioAHE2HAbAAzPhsBjSJGhcKRGLxARJFLpLI5fIo0ENAC+RElYJlEJwZFSZAxDQhUJhcMRqPRDz0NnsTjcHi8vn8bCCnIiUTiCX5TEF2TyBUKZDKFSqmKxZDqHs9jX+Ry6PQW2Guww640mOpmSjm2GDS3W2F+JxIuymPt9FQDJ1wFwuR1woeeJHuGd9TyGr3eJE+31+/0BZf0IIUTb0OLxBKJJLJlOpdIZbGZvFZ7It3OtfOSdsyDpFYv00vFQA=).

Reply:
I can do it on a subsequent PR.